switchbot/adv_parsers/blind_tilt.py

```diff
@@ -0,0 +1,25 @@
+"""Library to handle connection with Switchbot."""
+from __future__ import annotations
+
+
+def process_woblindtilt(
+    data: bytes | None, mfr_data: bytes | None, reverse: bool = True
+) -> dict[str, bool | int]:
+    """Process woBlindTilt services data."""
+
+    if mfr_data is None:
+        return {}
+
+    device_data = mfr_data[6:]
+
+    _tilt = max(min(device_data[2] & 0b00111111, 100), 0)
```

Comment:
I think this should be:
```suggestion
    _tilt = max(min(device_data[2] & 0b01111111, 100), 0)
```
Otherwise it limits the tilt value to 0-63

Reply:
This could explain some issues I was having =)